packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -121,6 +121,7 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   const adapter = useAdapter();
 
   useEffect(() => {
+    // Initial data should be always fetched by the composite(or external caller) instead of the adapter
     adapter.fetchInitialData();
```

Comment:
(feel free to disregard)
thoughts on "loadChatContent"?

"initial" is tough because it denotes some sense of "time" like only once and at the beginning.
Do we call this once at the beginning or can we call this at  anytime?

Reply:
We only call this once when we want to start the chat, and it wouldn't be called again (there is fetch message function to serve this purpose)